assets/constants.ts

```diff
@@ -5244,6 +5244,10 @@ export = {
 
   "NUM_QUESTIONS_PER_PAGE": 10,
 
+  // The default number of opportunities to show on the contributor dashboard
+  // page.
+  "OPPORTUNITIES_PAGE_SIZE": 10,
```

Comment:
Why did this number decrease from 20?

Reply:
Earlier we had an infinite scroll feature for presenting opportunities but we found that presenting multiple opportunities at a time was leading to a lag while writing suggestions on the text editor in the modal. This PR replaces infinite scroll with pagination and loads 10 opportunities per page to reduce/removes lag on the editor modal.

Reply:
Ah, that's much clearer (and now the constant makes a bit more sense, too). Thanks!